src/main/io/ledstrip.c

```diff
@@ -303,80 +283,70 @@ bool parseLedStripConfig(int ledIndex, const char *config)
             config++;   // skip separator
         }
         switch (parseState) {
-            case X_COORDINATE:
-                x = atoi(chunk);
-                break;
-            case Y_COORDINATE:
-                y = atoi(chunk);
-                break;
-            case DIRECTIONS:
-                for (char *ch = chunk; *ch; ch++) {
-                    for (ledDirectionId_e dir = 0; dir < LED_DIRECTION_COUNT; dir++) {
-                        if (directionCodes[dir] == *ch) {
-                            direction_flags |= LED_FLAG_DIRECTION(dir);
-                            break;
-                        }
+        case X_COORDINATE:
+            x = atoi(chunk);
```

Comment:
Codacy found an issue: [Unless checked, the resulting number can exceed the expected range (CWE-190).  If source untrusted, check both minimum and maximum, even if the input had no minus sign (large numbers can roll over into negative number; consider saving to an unsigned value if that is intended). ](https://app.codacy.com/gh/emuflight/EmuFlight/pullRequest?prid=6307959)